### ISSUE TITLE
Allow table cell to contain an array (list) of values 

### ISSUE THREAD

**Issue by jfcorbett** [startables/startable-standard#9]
Allow cells to contain a list of values; possibly to be immediately unpacked upon reading. 

This could be indicated in the column's unit field, e.g. `m[]` for whitespace-delimited values with unit `m`. Cell could then contain e.g. `1.3 4.5 22.22` or `1.2\n3.45\n6.789`.

Could add support for other delimiters e.g. `m[,]` for comma-separated, though this would require user to ensure that there is no collision with the file-level separator, e.g. commas or semicolons in CSV files. 

Potential semi-solution: in reader, unpack these lists as if they were rows (with non-list cols repeated). Requires all lists to have same number of elements. (else error)

In other words, this is shorthand for writing multiple rows as if they were "collapsed".

However this doesn't help when writing... they will remain exploded, not collapsed.
Could have flag that forces writer to collapse rows with? Investigate whether this this possible without crazy complexity

Possibly limit this "list" functionatlity to e.g. Word reader-writer, because other file formats wouldn't really need this in practice

**Comment by jfcorbett** [comment]
Migrated this from a legacy internal issue tracker.

**Comment by jfcorbett** [comment]
This issue is moribund. No specific proposal has been put forth that garners broad support and addresses this issue's many inherent pitfalls. Closing for now.
